README.md

```diff
@@ -1 +1,68 @@
-# Weather-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project: Weather-App
+
+> A Front-end web application that fetches the current weather data about a specified location/region/country using the http://api.openweathermap.org/ API.
+
+![screenshot](./src/images/Screenshot.png)
+
+## Live demo
+
+🔗 [visit the page]()
+
+## Built With
+
+- Javascript (ES6)
+- HTML / CSS
+- Font awesome icons
+- Webpack
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+In order for ```Weather-App``` to work locally on your machine, you need these installations:
+- Nodejs and npm
+- Web Browser
+
+### Install
+
+1) Open the Terminal
+2) Run
+
+```sh
+git clone https://github.com/RICKCOYL/Weather-App.git
+```
+
+3) Run ```cd Weather-App``` to enter the main directory
+4) Run ```npm install```
+5) Run ```npm run build```
+6) Run ```cd dist/```
+8) Open the ```index.html``` in your browser
+9) Search for weather data in any region/city/country of choice
+`
+
+## Author
+
+👤 **RICK OBURU**
+
+- Github: [https://github.com/RICKCOYL](https://github.com/RICKCOYL)
+- Twitter: [https://twitter.com/rickcoyl](https://twitter.com/rickcoyl)
+- Linkedin: [https://www.linkedin.com/in/rick-oburu/](https://www.linkedin.com/in/rick-oburu/)
+
```

Comment:
## Improve social links 
- [ ] There is a better way to showcase these social links to appear much neater. Kindly  have a look and tell me what you think down below in my suggestions 👇 

```suggestion
- Github: [@RICKCOYL](https://github.com/RICKCOYL)
- Twitter: [@rickcoyl](https://twitter.com/rickcoyl)
- Linkedin: [Rick Oburu](https://www.linkedin.com/in/rick-oburu/)

```